lib/graph3d/Settings.js

```diff
@@ -250,6 +250,15 @@ function setOptions(options, dst) {
   setSpecialSettings(options, dst);
 }
 
+/**
+ * Merge fields from src to dst
+ */
+function mergeFields(src, dst, fields) {
```

Comment:
I think something like this already exists in the utils [here](https://github.com/p-a/vis/blob/d4a8e878a36f26ea539a2bd15060eecb542a5bfa/lib/util.js#L198).